.gitlab-ci.yml

```diff
@@ -0,0 +1,15 @@
+image: ubuntu:bionic
+
+variables:
+  GRADLE_VERSION: "4.10.3"
+
+before_script:
+  - apt-get update && apt-get -y upgrade
+  - apt-get -y install openjdk-11-jdk gradle
+  - gradle -c empty-settings.gradle wrapper --gradle-version=$GRADLE_VERSION --info --stacktrace
+  - chmod +x ./gradlew
```

Comment:
This line isn't necessary; `gradlew` runs without.

Reply:
OK.

Reply:
Removed.